apps/web/src/components/Menu/types.ts

```diff
@@ -0,0 +1,9 @@
+export type TMenuItem = {
+  label: string;
+  icon: React.ReactNode;
+  onClick: (value?: any) => Promise<void> | void;
+  rightElement?: React.ReactNode;
+  hasArrow?: boolean;
```

Comment:
maybe it's better to pass in `nextPage` or `link` or whatever here instead? so that the menu item component can decide for itself if it needs an arrow and where this arrow should lead to